bundler/lib/bundler/current_ruby.rb

```diff
@@ -82,7 +71,8 @@ def windows?
         RUBY_VERSION.start_with?("#{version}.")
       end
 
-      KNOWN_PLATFORMS.each do |platform|
+      all_platforms = PLATFORM_MAP.keys << "maglev"
```

Comment:
I mentioned this in the commit but to highlight it here:

> The only change worth highlighing is the platform "maglev".
  It was not part of the supported platform of dependencies,
  so calling `gem 'foo', plaftorm: 'maglev'` would not work.
  However, it was supposed to according to https://github.com/Edouard-chin/rubygems/commit/45ec86e2e5288cc0302783a1c9ad9c5a15b4a6a2.
  That's why it was possible to do `Bundler.current_ruby.maglev?` or
  `Bundler.current_ruby.maglev_30?`.

> I didn't change the current behaviour and maglev is not supported,
  though I kept the `*maglev` methods as I believe CurrentRuby is
  public API.

Reply:
We should probably define `maglev` helpers separately and add a deprecation warning to them, but we can do that on a separate PR.

Reply:
Make sense 👍. I'll do a follow up, I think it will be easier.